Automattic/jetpack
# Show Browse sites when using nav unification and wp-admin interface

Slack: p1709660486029189-slack-C06DN6QQVAQ

## Proposed changes:

| Proxied | Unproxied |
| --- | --- |
| ![image](https://github.com/Automattic/jetpack/assets/402286/69dfc4ad-e015-4272-944a-34040ca39dd4) | ![image](https://github.com/Automattic/jetpack/assets/402286/bc7a578c-1385-4222-ac92-d0ef07e754ae) |


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Using a site with Classic interface and not `wpcom_classic_early_release`
* Go to `/wp-admin` while Proxied
* `< All Sites` should show on top of the menu
* Disconnect from the proxy
* Go to `/wp-admin`
* It should show nav unification instead of classic interface
* `Browse sites` should show on top of the menu
*
